src/services/devices.repository.ts

```diff
@@ -105,6 +105,34 @@ export class DevicesRepository {
         DevicesRepository.commands.next({ type: 'activate-scene', uid: this.uid, deviceIds, group, deactivate });
     }
 
+    sendToDock(group: string, deviceIds: string[], { notifyClient = false, requestId }: { notifyClient?: boolean, requestId?: string } = {}) {
```

Comment:
why do we need to handle this differently? It seems like a normal state change... isn't it?

Reply:
there is no docked state. thats why im handling it like the scene command. we dont send a response back